plugin/trino-mysql-event-listener/src/main/java/io/trino/plugin/eventlistener/mysql/QueryDao.java

```diff
@@ -32,7 +32,7 @@ public interface QueryDao
             "  trace_token VARCHAR(255) NULL,\n" +
             "  remote_client_address VARCHAR(255) NULL,\n" +
             "  user_agent VARCHAR(255) NULL,\n" +
-            "  client_info VARCHAR(255) NULL,\n" +
+            "  client_info MEDIUMTEXT NULL,\n" +
```

Comment:
But how would perform migration - In case of older tables which has this column as a `VARCHAR(255)`

Reply:
I think for both this and the DB resource groups we rely on users manually managing the schema. There are no migrations.

Introducing migrations is a bigger task here and with how less often the changes happen here I think it's fine to skip for now.